monfrontend/app.py

```diff
@@ -130,6 +151,43 @@ def fetch_data(intervals):
         }
     return measurements_data
 
+@app.callback(
+    Output("machine-status", "children"), [Input("in-memory-storage", "data"),],
+)
+def live_status(data):
+    """
+    Generates live machine parameters as children of div 'machine-parameters'
+    """
+    measurements = list(influx.get_measurements())
+    # for now use all available measurements, instead data.keys?
+
+    children = []
+    for msmt in measurements:
+        children.append(
+            html.H6(f"MachineStatus:{1}", className="motor_status"),
```

Comment:
Here you will get as many MachineStatus strings with identical contents ("MachineStatus:1") as there are measurements in the InfluxDB. Is that intended?

Reply:
No that's not intended. Wrong Syntax/placeholder. Needs to be mapped to actual status values transmitted.

Reply:
Okay. The measurements to display (and stuff like the display name and unit) should be set in the configuration (coming soon-ish). How about we just drop the loop for now in these two places (this and line 182) and return just the static placeholders you implemented?